app/decorators/member_decorator.rb

```diff
@@ -1,21 +1,24 @@
+# frozen_string_literal: true
+
+# MemberDecorator
 class MemberDecorator < Draper::Decorator
   delegate_all
 
-  def worked_hours_of_three_last_months
+  def hours_worked_in_the_three_last_months(csv: false)
```

Comment:
naming -> `hours_worked_in_the_last_three_months`
(sinon ça se lit "heures travaillées sur les 3 mois derniers" (ou "sur les 3 derniers mois de..." selon le contexte) au lieu de "heures travaillées sur les 3 derniers mois")

Reply:
it is already fixed, no ?